CondFormats/DataRecord/interface/L1TMuonBarrelParamsRcd.h

```diff
@@ -5,11 +5,15 @@
 // Class  :     L1TMuonBarrelParamsRcd
 //
 // Author:      Giannis Flouris
+// Kalman Mod:  Panos Katsoulis
 // Created:
 //
 
 #include "FWCore/Framework/interface/EventSetupRecordImplementation.h"
 
 class L1TMuonBarrelParamsRcd : public edm::eventsetup::EventSetupRecordImplementation<L1TMuonBarrelParamsRcd> {};
 
+class L1TMuonBarrelKalmanParamsRcd
```

Comment:
Is there any reason why not to use a dedicated file for the new class (as it is done for L1TMuonBarrelParamsO2ORcd.h, for example)

Reply:
done, fixing commit is coming